pkgs/development/python-modules/anndata/default.nix

```diff
@@ -72,51 +72,31 @@ buildPythonPackage rec {
     pytest-xdist
     pytestCheckHook
     scikit-learn
-    zarr
+    scanpy
   ];
 
   # Optionally disable pytest-xdist to make it easier to debug the test suite.
   # Test suite takes ~5 minutes without pytest-xdist. Note that some tests will
   # fail when running without pytest-xdist ("worker_id not found").
   # pytestFlags = [ "-oaddopts=" ];
 
-  disabledTestPaths = [
-    # Tests that require scanpy, creating a circular dependency chain
-    "src/anndata/_core/anndata.py"
-    "src/anndata/_core/merge.py"
-    "src/anndata/_core/sparse_dataset.py"
-    "src/anndata/_io/specs/registry.py"
-    "src/anndata/_io/utils.py"
-    "src/anndata/_warnings.py"
-    "src/anndata/experimental/merge.py"
-    "src/anndata/experimental/multi_files/_anncollection.py"
-    "src/anndata/utils.py"
-  ];
+  preCheck = ''
+    export NUMBA_CACHE_DIR=$(mktemp -d);
+  '';
+
+  doCheck = false;
```

Comment:
perhaps i'm missing something... where did the tests go?

Reply:
Into passthru.tests (just below)